torchgeo/trainers/moco.py

```diff
@@ -62,6 +62,8 @@ def moco_augmentations(
             T.RandomGrayscale(weights=weights, p=0.2),
             # Not appropriate for multispectral imagery, seasonal contrast used instead
             # K.ColorJitter(brightness=0.4, contrast=0.4, saturation=0.4, hue=0.4, p=1)
+            K.RandomBrightness(brightness=(0.8, 1.0), p=0.8),
+            K.RandomContrast(contrast=(0.8, 1.0), p=0.8),
```

Comment:
I still need to look into the details of the implementations, but shouldn't this be `(0.6, 1.4)` and `p=1`?